setup.py

```diff
@@ -25,7 +25,7 @@
     install_requires=[
         "click >= 4.1",
         "sqlparse >= 0.1.19",
-        "psycopg2 >= 2.7.4",
+        "psycopg >= 3.0.9,<3.1.0",
```

Comment:
should change this to >= 3.0.10. I advice to drop the <3.1.0: there is now abundant literature advising against using an upper bound in dependencies.